src/relay/transforms/device_planner.cc

```diff
@@ -20,6 +20,10 @@
 /*!
  * \file src/relay/transforms/device_planner.cc
  * \brief Determines a unique \p SEScope to hold the result of every Relay sub-expression.
+ * This pass can be run multiple times, and can be run both before and after lowering.
+ *
+ * TODO(mbs): Rename SEScope |-> VirtualDevice, and use 'virtual device' (or just 'device')
```

Comment:
I think we can do this now.

Reply:
Lemme clear #9613 then it's a clean alpha conv with no semantic changes.